managed/services/server/server.go

```diff
@@ -35,18 +35,24 @@ import (
 	"github.com/pkg/errors"
 	"github.com/sirupsen/logrus"
 	"golang.org/x/crypto/ssh"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/backoff"
 	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/credentials/insecure"
 	"google.golang.org/grpc/status"
 	"google.golang.org/protobuf/types/known/durationpb"
 	"google.golang.org/protobuf/types/known/timestamppb"
 	"gopkg.in/reform.v1"
 
 	"github.com/percona/pmm/api/serverpb"
+	"github.com/percona/pmm/api/updatepb"
 	"github.com/percona/pmm/managed/models"
 	"github.com/percona/pmm/managed/utils/envvars"
 	"github.com/percona/pmm/version"
 )
 
+const updaterSocketPath = "/srv/pmm-server-upgrade.sock"
```

Comment:
Do we want to put the socket in `/srv`? How about storing it in `/tmp` directory?

@BupycHuk what is your opinion in view of our HA spec?

Reply:
`/srv` directory is already mounted, so we can reuse it to set up the second container without re-running PMM Server container